scripts/puppeteer.server.js

```diff
@@ -196,6 +205,20 @@ document.getElementById('switch-microphone').addEventListener('change', async ()
   }
 });
 
+document.getElementById('switch-speaker').addEventListener('change', async () => {
+  try {
+    const switchSpeakerSelect = document.getElementById('switch-speaker');
+    const speakerID = switchSpeakerSelect.value;
+    const nodeAudio = document.getElementById('remote-audio');
+    if (nodeAudio?.setSinkId) {
```

Comment:
please move the `if` after the `await`, so if the action fails, we do not change the sink